src/PlaygroundWithGit.jl

```diff
@@ -31,10 +31,16 @@ function loan_interest_calculator()
 
   a = 0.0
   b = 1.0
+  num_iter = 0
   fa, fb = f(a), f(b)
   x = (a + b) / 2
   fx = f(x)
-  while abs(fx) > 1e-6
+  while abs(fx) > 1e-6 || (b - a) > 1
```

Comment:
It is better to use 1e-6 as interval minimum, so it is consistent, and also small.